### ISSUE TITLE
Configuring the O3DE from Github fails when using CMake below version 3.24.0

### ISSUE THREAD

**Issue by LB-KacperKapusta** [o3de/o3de#12981]
**Describe the bug**
Using CMake below version 3.24.0 while configuring the O3DE from Github (both Source and Pre-built SDK) results in incomplete configuration with the following error:
```
CMake Error at C:/builds/cmake-3.22.3-windows-x86_64/share/cmake-3.22/Modules/ExternalProject.cmake:2806 (message):
  At least one entry of URL is a path (invalid in a list)
Call Stack (most recent call first):
  C:/builds/cmake-3.22.3-windows-x86_64/share/cmake-3.22/Modules/ExternalProject.cmake:3716 (_ep_add_download_command)
  CMakeLists.txt:15 (ExternalProject_Add)
-- Configuring incomplete, errors occurred!
See also "C:/o3de/build/windows/_deps/rga-subbuild/CMakeFiles/CMakeOutput.log".
CMake Error at C:/builds/cmake-3.22.3-windows-x86_64/share/cmake-3.22/Modules/FetchContent.cmake:1075 (message):
  CMake step for rga failed: 1
Call Stack (most recent call first):
  C:/builds/cmake-3.22.3-windows-x86_64/share/cmake-3.22/Modules/FetchContent.cmake:1216:EVAL:2 (__FetchContent_directPopulate)
  C:/builds/cmake-3.22.3-windows-x86_64/share/cmake-3.22/Modules/FetchContent.cmake:1216 (cmake_language)
  C:/builds/cmake-3.22.3-windows-x86_64/share/cmake-3.22/Modules/FetchContent.cmake:1259 (FetchContent_Populate)
  cmake/3rdParty/FetchRGA.cmake:22 (FetchContent_MakeAvailable)
  cmake/3rdParty.cmake:368 (include)
  CMakeLists.txt:35 (include)
-- Configuring incomplete, errors occurred!
See also "C:/o3de/build/windows/CMakeFiles/CMakeOutput.log".
```
Please note that the [System Requirements](https://www.o3de.org/docs/welcome-guide/requirements/) point CMake version 3.21.7 or later as correct.
For further information refer to the attached CMakeOutput.log.

**Steps to reproduce**
1. Get the latest O3DE from Github (1e41006).
2. Open a command prompt in the directory where you set up the O3DE and run:
`python\get_python.bat`
3. In the same directory as Step 2 run:
`cmake -B build/windows -S . -G "Visual Studio 16" -DLY_3RDPARTY_PATH=C:\o3de-packages`

**Expected behavior**
Either configuring passes without issues for CMake 3.21.7 or later as described in [System Requirements](https://www.o3de.org/docs/welcome-guide/requirements/) or the mentioned System Requirements are updated to mention CMake 3.24.0.

**Actual behavior**
Configuring fails for CMake below version 3.24.0.

**Screenshots/Video**
![ConfiguringIncomplete](https://user-images.githubusercontent.com/86953659/199672747-04affbf9-68af-4abd-b645-5c32a7cb073f.PNG)

**Found in Branch**
Development (1e41006)

**Desktop/Device**
- Device: PC
- OS: Windows
- Version: 10
- CPU: AMD Ryzen 5 3600
- GPU: NVIDIA GeForce RTX 2060 SUPER
- Memory: 16GB

**Additional context**
[CMakeOutput.log](https://github.com/o3de/o3de/files/9926532/CMakeOutput.log)

**Comment by hultonha** [comment]
Fixed by https://github.com/o3de/o3de/pull/12977

**Comment by LB-KacperKapusta** [comment]
Issue verified as fixed in Development (ea9f21c) using CMake 3.21.7.
